src/app/pages/rpc-sign-bip322-message/rpc-sign-bip322-message.tsx

```diff
@@ -29,13 +30,25 @@ function RpcSignBip322Message() {
     origin,
     message,
     address,
-    isLoading,
+    isLoading: signBip322MessageIsLoading,
     onUserApproveBip322MessageSigningRequest,
     onUserRejectBip322MessageSigningRequest,
   } = useSignBip322Message();
 
+  const location = useLocation();
   const { chain } = useCurrentNetwork();
 
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
I added this code as if users:
- start signing this message
- hit `X` to cancel and go back
- when they go back to the correct place now `signBip322MessageIsLoading` stays stuck as loading so they cannot retry